offscreen_render_target/program_OpenGL.cpp

```diff
@@ -0,0 +1,121 @@
+#include "program_opengl.hpp"
```

Comment:
I saw the same file in the depending samples. Please illuminate duplication and change the name of this file to remove OpenGL from the file name - just program.hpp.